README.rst

```diff
@@ -0,0 +1,162 @@
+.. image:: ./util/images/craychapel.png
+
+
+.. _chapelhome-readme:
+
+The Chapel Language
+===================
+
+Chapel is an emerging programming language designed for productive parallel computing at scale. Chapel's design and implementation have been undertaken with portability in mind, permitting Chapel to run on multicore desktops and laptops, commodity clusters, and the cloud, in addition to the high-end supercomputers for which it was designed. Chapel's design and development are being led by `Cray Inc.`_ in collaboration with academia, computing centers, and industry. See `chapel.cray.com`_ for more information.
+
+.. _Cray Inc.: http://www.cray.com/
+.. _chapel.cray.com: http://chapel.cray.com/
+
+This is the 1.12.0 release of the Chapel compiler, intended to give
+potential users a look at what we're doing and the opportunity to
+provide us with feedback.  See the LICENSE file for the release's
+licensing terms.
+
+
+Quick Start
+-----------
+
+The following instructions are designed to get you up and running with
+Chapel with a minimum of fuss.  Note that building and using Chapel in
+this mode disables enhanced runtime options and optional language
+capabilities in the interest of a simple and clean build.
+
+
+0) See ``doc/prereqs.rst`` for more information about packages you
```

Comment:
Will references like these hyperlink to the files in question either (a) in the html view of this page and/or (b) the github view of the page (once it's the top-level README?).  If I could only have one, I think I'd want (a).  Both would be cool.  Neither would be unfortunate.